Formula/rakudo.rb

```diff
@@ -0,0 +1,24 @@
+class Rakudo < Formula
+  desc "Production-ready, stable implementation of the Perl 6 language"
+  homepage "https://rakudo.org"
+  url "https://rakudo.perl6.org/downloads/rakudo/rakudo-2018.12.tar.gz"
+  sha256 "67bb02b9afd4f2a2a8542e25ce2691bc2d77864668f0fc82e1d39dea31a584fa"
+  head "https://github.com/rakudo/rakudo.git"
+
+  depends_on "nqp"
+
+  def install
+    system "perl", "Configure.pl",
+      "--backends=moar",
+      "--prefix=#{prefix}",
+      "--with-nqp=#{Formula["nqp"].bin}/nqp"
+    system "make"
+    system "make", "install"
+    bin.install "tools/install-dist.p6" => "perl6-install-dist"
```

Comment:
Can you comment: if upstream does not wish to install this under `bin` (and thus put it in the user's `PATH`), why would we force this (and rename the tool)? Do other distros/vendors do this?

Reply:
Yes, on Void Linux, we have `perl6-install-dist` to support installing Perl 6 modules with xbps. Note the file `install-dist.p6` doesn't get installed anywhere on disk unless we manually place it into `bin`.

- https://github.com/void-linux/void-packages/commit/afdfdf99912399c8a634d72d2ee741813e68a24a
- https://github.com/void-linux/void-packages/commit/4cd7dba0bcf98d34dd719bc8f53418bb1e335f8c